src/routes.py

```diff
@@ -13,7 +15,8 @@
 @bp.route("/index")
 def index() -> str:
     """Use this view function to check whether Flask is installed properly."""
```

Comment:
Since this route is used more than just checking Flask installation, let's update docstring. We can use an approach we applied for `registration` route, like
```python
"""Handle index page."""
```